src/Schema/Table.php

```diff
@@ -346,6 +350,47 @@ public function addColumn($name, $typeName, array $options = [])
         return $column;
     }
 
+    /** @return array<string, string> */
+    final public function getRenamedColumns(): array
+    {
+        return $this->renamedColumns;
+    }
+
+    /**
+     * @throws Exception
+     * @throws SchemaException
+     */
+    final public function renameColumn(string $oldName, string $newName): Column
+    {
+        $oldName = $this->normalizeIdentifier($oldName);
+        $newName = $this->normalizeIdentifier($newName);
+
+        if ($oldName === $newName) {
+            throw new Exception(sprintf(
+                'You are trying to rename the column %s but you didn\'t change it\'s name',
```

Comment:
```suggestion
                "You are trying to rename the column %s but you didn't change its name",
```

Also, looking at the project, we very rarely address the user directly. I also think that the message should mention the name in case many columns are renamed. Also, you should probably throw a `LogicException` here as it is not meant to be caught.